webapp/app/js/perf.js

```diff
@@ -297,6 +334,12 @@ perf.factory('math', [ function() {
       return Math.sqrt(
         values.map(function (v) { return Math.pow(v - avg, 2); })
           .reduce(function (a, b) { return a + b; }) / (values.length - 1));
+    },
+
+    trimFloat: function(number) {
+      if (number === undefined)
+        return "notReady";
```

Comment:
This doesn't seem like a great return value. I think we should just return undefined here, and maybe log an error.